orga/tests/acceptance/student-list-test.js

```diff
@@ -0,0 +1,98 @@
+import { module, test } from 'qunit';
+import { currentURL, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+import {
+  createUserWithMembershipAndTermsOfServiceAccepted,
+  createUserMembershipManagingStudents
+} from '../helpers/test-init';
+
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Acceptance | Student List', function(hooks) {
+
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  let user;
+
+  module('When user is not logged in', function() {
+
+    test('it should not be accessible by an unauthenticated user', async function(assert) {
+      // when
+      await visit('/eleves');
+
+      // then
+      assert.equal(currentURL(), '/connexion');
+    });
+  });
+
+  module('When user is logged in', function() {
+
+    module('When organization is not managing students or is not SCO', function() {
```

Comment:
Je trouve le ``or is not SCO`` un peu perturbant. 
Est-ce que la seule condition dans l'implémentation ne vient pas du ``isManagingStudents`` ?

Reply:
non il faut que ce soit les deux